frontend/src/app/search/page.tsx

```diff
@@ -29,6 +37,26 @@ export default async function Search() {
   const initialSearchResults = await fetchSearchOpportunities(searchFetcher);
   return (
     <>
+      {/* TODO: i18n */}
+      <PageSEO
+        title="Search Funding Opportunities"
+        description="Try out our experimental search page."
+      />
+
+      <GridContainer>
+        {/* TODO: BetaAlert Breadcrumbs */}
+        {/* <BetaAlert /> */}
+        {/* <Breadcrumbs breadcrumbList={SEARCH_CRUMBS} /> */}
+        <h1 className="margin-0 tablet-lg:font-sans-xl desktop-lg:font-sans-2xl">
+          Search funding opportunities
+        </h1>
+        <p className="tablet-lg:font-sans-lg line-height-sans-3 usa-intro margin-top-2">
+          We’re incrementally improving this experimental search page. How can
+          we make it easier to discover grants that are right for you? Let us
+          know at <a href="mailto:simpler@grants.gov">simpler@grants.gov</a>.
+        </p>
+      </GridContainer>
+
```

Comment:
👍  Yea this file is all server rendered markup - since this is content is static this is the right place to put it!